src/molecules/Toggle/ToggleExample.jsx

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
OCD comments:
- please remove empty new line at the top of the file
- please make sure there's one extra line at the end of the file

More reasonable comments:
- I like your intentional separation here, but it'd be better, I think, if you just defined this `ToggleExample` object from your `index.js` file